myria/query.py

```diff
@@ -0,0 +1,71 @@
+import time
+import requests
+from . import MyriaRelation
```

Comment:
Please avoid `from .. import ..`. It leads to circular import issues later and is thus generally discouraged.